test/stripe/charge_test.rb

```diff
@@ -40,6 +40,16 @@ class ChargeTest < Test::Unit::TestCase
       assert charge.is_a?(Stripe::Charge)
     end
 
+    context ".capture" do
+      should "capture the charge" do
+        charge = Stripe::Charge.capture("ch_123", amount: 100)
```

Comment:
minor but I wish we moved to the nested hash for params. I think it's a lot easier to comprehend, and the docs require that.

Reply:
That's a good point. I don't mind changing the linter to make the library's code consistent with the style in our docs.